go-controller/pkg/ovn/service.go

```diff
@@ -70,6 +70,9 @@ func (ovn *Controller) syncServices(services []interface{}) {
 	// are stale.
 	for _, protocol := range []string{TCP, UDP} {
 		loadBalancer := ovn.getLoadBalancer(kapi.Protocol(protocol))
```

Comment:
Perhaps more idomatic to return an error from getLoadBalancer() if out == "" and then check err != nil here?